geometry2/860b866/860b866.bug

```diff
@@ -86,4 +86,4 @@ fault-codes:
 failure-codes:
   - SOFTWARE:BEHAVIORAL
   - SYSTEM:BEHAVING-INCORRECTLY
-  - SYSTEM:MODEL-REALITY-DIVERGENCE
+  - SYSTEM:MONITORING
```

Comment:
I don't quite see the case for MONITORING on this one. This bug leads to a bad vector translation in geometry2, which fits best with our BEHAVING-INCORRECTLY label. I suggest that we drop the MONITORING label.

Reply:
From the meeting: Let's keep both labels. MONITORING failures would be observed by the user when using, e.g., rviz. BEHAVING-INCORRECTLY would occur in all places where this transformation is used.